ambry-store/src/test/java/com.github.ambry.store/HardDeleterTest.java

```diff
@@ -256,13 +265,13 @@ public MessageInfo getMessageInfo(Read read, long offset, StoreKeyFactory factor
       tokenMovedForward = index.hardDelete();
       Assert.assertTrue(tokenMovedForward);
 
-      // indexes: [1 2] [3 4] [3d 5*] [6 7] [2d 6d] [8 9] [1d 10d] [8]
+      // indexes: [1 2] [3 4] [3d 5] [6 7] [2d 6d] [8 9*] [1d 10d] [8]
```

Comment:
Nope. nothing has changed. But the comment was wrong. So I fixed it. @pnarayanan can you confirm if I am doing it right